openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClassesWorkloadTest_Softmx_Increase_JitAot.java

```diff
@@ -49,40 +49,34 @@
  *  
  *  Implementation steps
  *  ~~~~~~~~~~~~~~~~~~~~
- *  1) Start 2 Jvms that run workload processes with options "-XX:SharedCacheHardLimit=20m, -Xscmxsoftmx=5m 
- *     -Xshareclasses:addtestjithints,verbose -Xscminaot30k -Xscmaxaot40k 
- *     -Xscminjitdata30k -Xscmaxjitdata40k -Xjit:count=1". 
+ *  1) Start 2 Jvms that run workload processes with options "-XX:SharedCacheHardLimit=20m, 
+ *     -Xshareclasses:addtestjithints,verbose-Xscmaxaot10k -Xscmaxjitdata10k -Xjit:count=0". 
  *     
  *     Both Jvms run mini-mix load tests, generate aot and jit data, and fill up the aot 
  *     and jit space in the cache. 
  *     
- *  2) Once Jvm1 and Jvm2 finishes, check cache stats to ensure cache 
- *     is soft full. Fail if it isn't. Also make sure the AOT and JIT space is full. 
+ *  2) Once Jvm1 and Jvm2 finishes, check that the AOT and JIT space is full. 
  *     Also, record the cache stats at this point, which should give us the number of bytes
  *     written in AOT and JIT data space. 
  *     
  *  3) Start Jvm3 with with options to increase softlimit of the cache and size of the aot and jit data spaces: 
  *  	 "-Xshareclasses:adjustsoftmx=10m,adjustminaot=50k,adjustmaxaot=60k,
  *     adjustminjit=50k,adjustmaxjit=60k". Check cache stats to ensure cache's limits has been increased.
```

Comment:
I see you removed CACHESIZE_SOFTLIMIT_INCREASED, adjustsoftmx=10m should also be removed from the comment here

Reply:
Good Catch! Updated comment.